README.org

```diff
@@ -606,6 +607,23 @@ If =:session= is a remote file name that doesn't end in =.json=, e.g.
 the =jupyter kernel= command on the host. The local part of the session name
 serves to distinguish different remote sessions on the same host.
 
+*** Communicating with kernel (notebook) servers
+
+If =:session= is a TRAMP file name like =/jpy:localhost#8888:NAME= it is
+interpreted as corresponding to a connection to a kernel through a Jupyter
+notebook server located at =http://localhost:8888=. If =NAME= is a kernel ID
+corresponding to an existing kernel on a server, e.g.
+=/jpy::161b2318-180c-497a-b4bf-de76176061d9=, then a connection to an existing
+kernel with the corresponding ID will be made. Otherwise, a new kernel will be
+launched on the server and =NAME= will be used as an identifier for the
+session.
+
+Note, the =:kernel= header argument must match the name of the kernelspec of
+the kernel being connected to when =NAME= is a kernel ID of an existing kernel.
```

Comment:
```suggestion
the kernel being connected to when =NAME= is a kernel ID of an existing kernel, otherwise a new kernel will be started.
```

Reply:
This is actually mentioned in the paragraph above and this section is about the `:kernel` header argument not the `:session` one. Do you want to take a crack at refactoring the above paragraph so that its clearer?